tests/integration/helpers/cluster.py

```diff
@@ -1448,10 +1466,9 @@ def create_dir(self, destroy_dir=True):
             self._create_odbc_config_file()
             odbc_ini_path = '- ' + self.odbc_ini_path
 
-        entrypoint_cmd = CLICKHOUSE_START_COMMAND
-
         if self.stay_alive:
             entrypoint_cmd = CLICKHOUSE_STAY_ALIVE_COMMAND
+            entrypoint_cmd = entrypoint_cmd.replace("{main_config_file}", self.main_config_name)
```

Comment:
The indent is wrong - you have to do this replacement always, not only if `stay_alive == True`.

Reply:
First replace is done at 1368 line: `entrypoint_cmd = entrypoint_cmd.replace("{main_config_file}", self.main_config_name)`
I added the second one here to fix `entrypoint_cmd` after `entrypoint_cmd = CLICKHOUSE_STAY_ALIVE_COMMAND`.